index.bs

```diff
@@ -2503,8 +2500,8 @@ must return [=/this=]'s [=connection/version=].
         [=/upgrade transaction=].
 </div>
 
-The <dfn attribute for=IDBDatabase>objectStoreNames</dfn> attribute's
-getter must return a {{DOMStringList}} associated with a [=sorted name list=] of the [=object-store/names=] of
+The <dfn attribute for=IDBDatabase>objectStoreNames</dfn> getter steps are to
+return a {{DOMStringList}} associated with a [=sorted name list=] of the [=object-store/names=] of
 the [=/object stores=] in [=/this=]'s [=object store set=].
```

Comment:
Preexisting issue: it's not clear whether this returns the same DOMStringList each time, or a new DOMStringList. (The former would certainly be better.)

Reply:
Fixed - sadly by inserting "new" in each such site, at least per Chrome's impl. I don't see WPT for this, so it might be web compatible to change. It gets weird for IDBTransaction's `objectStoreNames` property since that can change over time. In any case this isn't a method that's ever going to be a performance bottleneck so it's not a big deal.